### ISSUE TITLE
launcher update correct architecture

### ISSUE THREAD

**Issue by david79111** [FCL-Team/FoldCraftLauncher#679]
### Before making issue / 在发布提问之前...

- [X] I understand this is a request, not a wish.我明白这是在提出请求而不是在许愿。

### Description / 描述

Either the launcher should recognize the correct apk architecture of the device or the user should be able to select the correct architecture once so the launcher will always download the correct architecture

### Why / 原因

It saves ressources when downloading the correct architecture over the universal architecture apk.

**Comment by 199825443** [comment]
可以直接在github下载对应架构啊

**Comment by david79111** [comment]
yeah I know I can download it from github, but it wouldnt hurt to implement it in the launcher if theres already a built in update method
